src/helpers/get-teams.js

```diff
@@ -16,19 +19,24 @@ export async function getTeams(serviceName, logger) {
   }
 
   const service = await fetchService(serviceName)
-  if (!service?.teams) {
-    logger.info(
-      {
-        event: {
-          ...logger.bindings().event,
-          outcome: 'No alert sent',
-          reason: `service ${serviceName} was not found`
-        }
-      },
-      `service ${serviceName} was not found`
-    )
-    return []
+  if (service?.teams) {
+    return service.teams
   }
 
-  return service.teams
+  if (serviceName.startsWith('cdp-')) {
+    return Promise.resolve(platform)
```

Comment:
FYI theres no need for a `Promise.resolve()` here you should just return for a `resolve` or throw for a `reject`. The entire function is a `Promise` - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function#description